ecl/eclagent/eclagent.cpp

```diff
@@ -2502,6 +2502,8 @@ bool EclAgent::checkPersistUptoDate(IRuntimeWorkflowItem & item, const char * lo
         errText.appendf("Rebuilding PERSIST('%s'): Saved CRC isn't present", logicalName);
```

Comment:
I don't know why the CRC would be missing, but I agree with @jamienoss, if REFRESH(false) is used, then must still check if file is present, but should ignore missing CRC.